Examples/E2/Notification.cs

```diff
@@ -13,26 +13,43 @@ public class SmsNotification : INotification
     public void Send(string message) => Console.WriteLine($"SMS sent: {message}");
 }
 
-#region Thid block of code is not in your code base, it is a library(3rd party)
+
+#region This block of code is not in your code base, it is a library(3rd party)
 public class SlackNotification
 {
     public void PostMessageToChannel(string channel, string message)
     {
         Console.WriteLine($"Message sent to Slack channel {channel}: {message}");
     }
-} 
+}
 #endregion
+// We Should Create New Class And Inherit From SlackNotification And Impelement INotification InterFace
+// Then We Can Call PostMessageToChannel In INotification Send Method
+// For Channel Parameter We Can Get that from constructor or Hardcode That
+public class CustomSlackNotification : SlackNotification, INotification
```

Comment:
sometimes in some cases, we cant inherit of 3rd party. 
for example: 
multiple inheritance, or slack notification is modified by sealed keyword.

consider that your solution is good enough here, but bot in other cases.

Reply:
It was a beautiful point👌
So what we have to do when working with seald classes in 3rd party packages?